config/StoredProcedures/CreateUser.sql

```diff
@@ -15,7 +15,7 @@ union
 select GroupName as name from UserGroups) as s1 where s1.name=_Username;
 
 IF @unique<=0 THEN
-	INSERT INTO Users (Username, FirstName, LastName, Email, PermsID, HP, US, DateCreated)  VALUES(_Username,_FirstName,_LastName,_Email,@pmid,_HP,_US,_DateCreated);
+	INSERT INTO Users (Username, FirstName, LastName, Email, PermsID, HP, US, DateCreated)  VALUES(_Username,_FirstName,_LastName,_Email,@pmid,_HP,_US,_DateCreated, (1));
```

Comment:
We should always keep the column names, even if we are using all columns. Explicit is better than implicit. Keeping the column names in creates self-documenting code without needing to reference the table setup to know what a column should be.